### ISSUE TITLE
Lighthouse throws warnings about accessibility issues on pfe-accordion

### ISSUE THREAD

**Issue by starryeyez024** [patternfly/patternfly-elements#458]
Failing Elements (per lighthouse)

```
<button aria-expanded="false" role="tab">Why do wizards need money if they could just create it?</button>
```

1. [role]s are not contained by their required parent element
    - Some ARIA child roles must be contained by specific parent roles to properly perform their intended accessibility functions. [Learn more.](https://dequeuniversity.com/rules/axe/3.2/aria-required-parent)
2. [role] values are not valid
    - ARIA roles must have valid values in order to perform their intended accessibility functions. [Learn more.](https://dequeuniversity.com/rules/axe/3.2/aria-roles)
3. [aria-*] attributes do not have valid values
    - Assistive technologies, like screen readers, can't interpret ARIA attributes with invalid values. [Learn more.](https://dequeuniversity.com/rules/axe/3.2/aria-valid-attr-value)

Page: https://www-admin.corp.redhat.com/en/campaign/web-component-test

Report: (had to change extension from .json to .txt)
[www-admin.corp.redhat.com-20190630T215646.txt](https://github.com/patternfly/patternfly-elements/files/3352082/www-admin.corp.redhat.com-20190630T215646.txt)

**Comment by castastrophe** [comment]
Some additional context to these errors: https://www.w3.org/TR/wai-aria-1.1/#tab

It appears that `role="tab"` should be contained inside `role="tablist"`.

Per the `aria-expanded="true"` being invalid value, I am wondering if it's because true is quoted?  Here is what it lists as valid in the spec: https://www.w3.org/TR/wai-aria-1.1/#aria-expanded

**Comment by kylebuch8** [comment]
I think we might actually need to get away from a tablist and role="tab" and instead just use a button (like we're already doing) and a region (like we're already doing). 

We can remove the `role="tabpanel"` here
https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-accordion/src/pfe-accordion-panel.html#L1

And remove `role="tab"` here
https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-accordion/src/pfe-accordion-header.html#L1

I'd use this as a guide
https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html

**Comment by kelsS** [comment]
Hi @kylebuch8 I went through the issues with @grantsydney. The last attribute that also needed to be removed was `role="tablist"`.  https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-accordion/src/pfe-accordion.js#L42 

All her changes took the Lighthouse score from an 86 to a 93.  I'm compiling the before and after reports to document progress. I also tested the changes with a screen reader and the screen reader experience is very good.  Awesome work everyone! 🎉🎉🎉
